validate.py

```diff
@@ -125,6 +125,10 @@ def test_haproxy(self):
         list_dir = os.listdir(os.path.join(self.acserver_d, "haproxy"))
         self.assertEqual(1, len(list_dir))
 
+    def test_socat(self):
```

Comment:
Overall i think we need to move all of this somewhere else.
Or have a better understanding of what validate.py is